internal/tiltfile/k8s.go

```diff
@@ -92,12 +97,14 @@ func (s *tiltfileState) k8sResource(thread *starlark.Thread, fn *starlark.Builti
 	var yamlValue starlark.Value
 	var imageVal starlark.Value
 	var portForwardsVal starlark.Value
+	var extraPodLabelsVal starlark.Value
```

Comment:
if you make this a skylark.Sequence, does unpack args do the right thing by default?

Reply:
if I make it a `starlark.Sequence`, I get a panic:
```
panic: interface conversion: interface is nil, not starlark.Value [recovered]
	panic: interface conversion: interface is nil, not starlark.Value
```
I guess `starlark.Sequence` isn't a `starlark.Value`?

If I make it a `starlark.List`, I get a `log.Fatalf`, and the program dies!:
```
2019/01/31 14:07:40 internal error: invalid pointer type: *starlark.List
```

If I make it a `starlark.Iterable`, it works, but I'm not sure that's what we want since it also accepts non-list iterables, like dicts, which then gets you this confusing error if you give it a dict instead of a dict in a 1-element list:
```
pod labels elements must be dicts; got starlark.String
```

Reply:
allowing either dict or list makes this irrelevant, but that was still kinda surprising